src/runtimes/docker/utils/getPlatformString.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ContainerPlatform } from "../../../tasks/DockerBuildTaskDefinitionBase";
+
+/**
+ * This method parses the `platform` field in tasks.json
+ * @returns Platform string specified by `platform`
+ */
+export function getPlatformString(platform: ContainerPlatform): string | undefined {
```

Comment:
This method should be a DockerClientBase helper, not a generic runtime util as it's specifically generating a docker --platform argument. Also, it makes more sense to handle default values for os/architecture here than in the normalization function, as it makes the logic much simpler.

Reply:
See my other comment on using a custom argument method instead.

Reply:
@bwateratmsft @danegsta should we convert this into the the `withDockerPlatformArg` method that David mentioned below?